velox/serializers/PrestoSerializer.cpp

```diff
@@ -4265,6 +4265,26 @@ void PrestoVectorSerde::deserializeSingleColumn(
   *result = row->childAt(0);
 }
 
+void PrestoVectorSerde::serializeSingleColumn(
+    const VectorPtr& vector,
+    const PrestoOptions& opts,
+    memory::MemoryPool* pool,
+    std::ostream* output) {
+  const auto numRows = vector->size();
```

Comment:
Thanks for pointing it out, renamed the variable to size and also removed the condition that the input vector contain a single element since it was specific to our usecase in Presto.